.github/CONTRIBUTING.md

```diff
@@ -1,11 +1,9 @@
-Contributing
-------------
+## Contributing
```

Comment:
<img width="1055" alt="Screen Shot 2019-11-19 at 8 36 57 AM" src="https://user-images.githubusercontent.com/10549495/69126153-c5a59200-0aa7-11ea-8f9d-faa113d8081e.png">
It looks all right, please revert

Reply:
@EugeneHlushko this is the work of prettier + markdown lint😅